packages/lesmis/src/views/ExportView.js

```diff
@@ -90,15 +93,18 @@ const getChildren = ({
 export const ExportView = ({ viewProps, viewType, className }) => {
   const { getExtraExportViewProps, PageContainer, PageContent } = EXPORT_VIEWS[viewType];
   const exportViewProps = { ...viewProps, ...getExtraExportViewProps() };
-
+  const { selectedDashboard } = exportViewProps;
   const { dropdownOptions } = useDashboardDropdownOptions();
-  const profileDropDownOptions = dropdownOptions.filter(({ exportToPDF }) => exportToPDF);
-  const { exportableDashboards } = getExportableDashboards(profileDropDownOptions);
+  const profileDropDownOption = dropdownOptions.find(({ value }) => value === selectedDashboard);
+  const { exportableSubDashboards } = getExportableSubDashboards(profileDropDownOption);
 
   return (
     <Container className={className}>
-      {exportableDashboards?.map((subDashboard, index) => {
-        const isFirstPageProfile = index === 0;
+      {exportableSubDashboards?.map((subDashboard, index) => {
+        const isFirstPageProfile =
+          viewType === DASHBOARD_EXPORT_PREVIEW
+            ? selectedDashboard === 'profile' && index === 0
```

Comment:
```suggestion
            selectedDashboard === 'profile' && index === 0
```

We should show the entity profile in preview and export view?